config/sync/user.role.site_developer.yml

```diff
@@ -0,0 +1,150 @@
+uuid: d7aedf9d-28c2-4f44-86ff-4f15ea6b8a39
+langcode: en
+status: true
+dependencies: {  }
+id: site_developer
+label: 'Site Developer'
+weight: 0
+is_admin: null
+permissions:
+  - 'access administration pages'
+  - 'access config_pages overview'
+  - 'access content overview'
+  - 'access contextual links'
+  - 'access devel information'
+  - 'access entity usage statistics'
+  - 'access environment indicator'
+  - 'access file_browser entity browser pages'
+  - 'access files overview'
+  - 'access image_browser entity browser pages'
+  - 'access media overview'
+  - 'access media_browser entity browser pages'
+  - 'access patterns page'
+  - 'access site in maintenance mode'
+  - 'access site reports'
+  - 'access taxonomy overview'
+  - 'access user profiles'
+  - 'access video_browser entity browser pages'
+  - 'admin classes'
+  - 'admin display suite'
+  - 'admin fields'
+  - 'administer account menu items'
+  - 'administer account settings'
+  - 'administer admin menu items'
+  - 'administer block_content display'
+  - 'administer block_content fields'
+  - 'administer block_content form display'
+  - 'administer blocks'
+  - 'administer config_pages display'
+  - 'administer config_pages fields'
+  - 'administer config_pages form display'
+  - 'administer config_pages types'
+  - 'administer content types'
+  - 'administer crop'
+  - 'administer crop types'
+  - 'administer devel menu items'
+  - 'administer display modes'
+  - 'administer embed buttons'
+  - 'administer entity browsers'
```

Comment:
This is going to be removed after stanford_media update is in place

Reply:
GTK, will update then.